### PR TITLE
Organizers update

### PR DESCRIPTION
**Pull request by benabraham**
feel free to add more people, correct errors, …
#betterdonethanperfect

### CODE REVIEW

File: series/praha-pyvo/series.yaml [hunk 1]
```diff
@@ -12,7 +14,9 @@ description:
 #     Pro samostatné a nestíhače: Kdyz se Ostrovského ulice změní na chodník,
 #     jděte cca 50 m do kopce. Po pravé straně narazíte na poměrně velkou bránu do dvora.
   en: |
-    Prague Pyvo is traditionally held on the third Wednesday of each month.
+    Prague Pyvo is traditionally held on the third Wednesday of each month."
```
Comment (hroncok) [inline series/praha-pyvo/series.yaml:17]: Proč je tady ta uvozovka?

Comment (benabraham) [reply series/praha-pyvo/series.yaml:17]:
```suggestion
    Prague Pyvo is traditionally held on the third Wednesday of each month.
```

File: series/praha-pyvo/series.yaml [hunk 2]
```diff
@@ -13,6 +15,8 @@ description:
 #     jděte cca 50 m do kopce. Po pravé straně narazíte na poměrně velkou bránu do dvora.
   en: |
     Prague Pyvo is traditionally held on the third Wednesday of each month.
+    
+    Feel free to drop us and email at [praha@pyvo.cz](mailto:praha@pyvo.cz) if you want to have a talk.
```
Comment (clytaemnestra) [inline series/praha-pyvo/series.yaml:19]:
```suggestion
    Feel free to drop us an email at [praha@pyvo.cz](mailto:praha@pyvo.cz) if you want to have a talk.
```

File: series/praha-pyvo/series.yaml [hunk 3]
```diff
@@ -25,14 +29,11 @@ description:
...
-  - name: Jakub Vysoký
-    phone: 605 852 377
-    mail: jakub.vysoky@gmail.com
-  - name: Jiří Bartoň
-    mail: whiskybar@gmail.com
-  - name: Aleš Zoulek
-    phone: 604 332 515
-    mail: ales.zoulek@gmail.com
+  - name: Mia Bajić
+    mail: praha@pyvo.cz
```
Comment (clytaemnestra) [inline series/praha-pyvo/series.yaml:33]:
```suggestion
    mail: praha@pyvo.cz
    phone: 777 177 720
```

**Comment by aleszoulek** [comment]
😢

**Pull request merged**